packages/paste-core/components/visual-picker/src/VisualPickerCheckboxGroup.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import {InlineControlGroup} from '@twilio-paste/inline-control-group';
+import {type CheckboxProps} from '@twilio-paste/checkbox';
+
+import {VisualPickerCheckboxContext} from './VisualPickerContext';
+import {type VisualPickerCheckboxGroupProps} from './types';
+
+export const VisualPickerCheckboxGroup = React.forwardRef<HTMLFieldSetElement, VisualPickerCheckboxGroupProps>(
+  (
+    {
+      orientation = 'vertical',
+      children,
+      element = 'VISUAL_PICKER_CHECKBOX_GROUP',
+      disabled = false,
+      errorText,
+      isSelectAll = false,
+      name,
+      onChange,
+      ...props
+    },
+    ref
+  ) => {
+    const onChangeHandler = React.useCallback(
+      (event: React.ChangeEvent<HTMLInputElement>): void => {
+        if (onChange != null) {
+          onChange(event.target.checked);
```

Comment:
observation(?): If I'm not mistaken, this goes to show that no one uses the checkboxgroup on change (where this came from) because, I think, this is kinda of useless. If we only pass you a boolean to the group change handler, it doesn't tell me anything, which one was checked? What am I supposed to do with that. I think we made a boo boo potentially... 😟

Reply:
removed checkbox group on change